### PR TITLE
Updated packages, alter hero image config

### PR DESCRIPTION
**Pull request by DamienBurkeWN**
### Setup
Checkout this branch and run a drush config:import

### Testing
1. Create a new service landing page. Verify there is now a Hero Image tab, but don't set one (and leave the empty fields in place, don't use the Remove button). Save in published state. Verify it allows the save without providing an image.
2. View your page in the frontend and verify it displays normally.
3. Edit the page in the CMS and add a hero image and save in published state.
4. View the page in the frontend and verify it now displays a hero image.
5. Edit a service page and verify the icon field is now the first one, as it is on service landing pages.

### CODE REVIEW

File: backend/src/config/sync/core.entity_form_display.node.service_landing_page.default.yml [hunk 1]
```diff
@@ -62,91 +61,107 @@ third_party_settings:
...
       format_settings:
-        id: ''
         classes: ''
-        description: ''
+        id: ''
         formatter: closed
+        description: ''
         required_fields: true
-      label: 'Top Services'
+    group_hero_image:
```
Comment (DamienBurkeWN) [inline backend/src/config/sync/core.entity_form_display.node.service_landing_page.default.yml:150]: New hero image tab added so the hero image setting is in its own tab and doesn't bother anybody who doesn't specifically need to set one

File: backend/src/config/sync/core.entity_form_display.node.service_page.default.yml [hunk 2]
```diff
@@ -32,99 +32,99 @@ third_party_settings:
   field_group:
     group_main:
       children:
+        - field_service_icon
```
Comment (DamienBurkeWN) [inline backend/src/config/sync/core.entity_form_display.node.service_page.default.yml:35]: icon moved to top field to be consistent with service landing page config

File: backend/src/config/sync/field.field.paragraph.hero_image.field_image.yml [hunk 3]
```diff
@@ -11,8 +11,8 @@ field_name: field_image
 entity_type: paragraph
 bundle: hero_image
 label: Image
-description: 'A large image to use as the background of the title, body and call to action'
-required: true
+description: 'A large image to use as the background of the title, body and call to action. If no image is set there will be no hero image used on the page and the normal service landing page layout will apply.'
+required: false
```
Comment (DamienBurkeWN) [inline backend/src/config/sync/field.field.paragraph.hero_image.field_image.yml:15]: Image no longer required, so an empty hero image paragraph is now acceptable and the emptiness is dealt with in the graphql resolution code.

File: backend/src/modules/nc_system/src/Entity/Node/ServiceLandingPage.php [hunk 4]
```diff
@@ -122,7 +122,8 @@ protected function getHeroImage(): ?HeroImage
     /* @var $heroImages array<\Drupal\nc_system\Entity\Paragraph\HeroImage> */
     $heroImages = $heroImageField->referencedEntities();
     if (!empty($heroImages)) {
-      return $heroImages[0];
+      $heroImage = $heroImages[0];
+      return !empty($heroImage->values['field_image']) ? $heroImage : null;
```
Comment (DamienBurkeWN) [inline backend/src/modules/nc_system/src/Entity/Node/ServiceLandingPage.php:126]: Detect the lack of an image within the hero image paragraph and return null if so for the entire thing, so no need to change schema and frontend.

**Comment by chrisrhymes** [comment]
When I try and save the new service landing page there is a validation error with Access -> Service area as it's empty, but the console error says it's not focusable.

**Comment by chrisrhymes** [comment]
> When I try and save the new service landing page there is a validation error with Access -> Service area as it's empty, but the console error says it's not focusable.

It is either Service area, or the Sections. 
`An invalid form control with name='field_sections[0][subform][field_pages][0][target_id]' is not focusable.`

**Review by chrisrhymes — APPROVED** [review]
Tried it out with the patch and all is now working as expected 👍

**Pull request merged**